zigen-project/zen
# Add app launcher icons

## Summary
Add app launcher icons on the menu bar.

(Note that this change assumes that the path to `google-chrome-stable` and `weston-terminal` executables are properly set in the user's environment). 

## How to check behavior
- Confirm that the launcher icons for chrome, blender and terminal are shown in the bottom-left corner of the display.
- Click each icon and see that the corresponding app starts.
